client/app/components/editor/editorView.html

```diff
@@ -1,13 +1,31 @@
-<div class="container">
+<div class="editor">
 
   <h3>{{ title }}</h3>
   <p>{{ prompt }}</p>
   <p ng-repeat="example in examples">{{ example }}</p>
+  <p>{{ numCorrect }}</p>
+  <ul>
+    <li class="correct" ng-repeat="detail in corrDetails track by $index">{{ detail }}</li>
+    <li class="incorrect" ng-repeat="incorrDetail in incorrDetails track by $index">{{ incorrDetail }}</li>
```

Comment:
Ng-repeat doesn't like duplicate key values apparently, so by $index solves for that using an element index. Hope that makes sense!



mark macdonald
415.278.1944
markm.ac

On Wed, Mar 18, 2015 at 8:16 PM, Chris Rinaldi <notifications@github.com>
wrote:

>>  
>>    <h3>{{ title }}</h3>
>>    <p>{{ prompt }}</p>
>>    <p ng-repeat="example in examples">{{ example }}</p>
>> +  <p>{{ numCorrect }}</p>
>> +  <ul>
>> +    <li class="correct" ng-repeat="detail in corrDetails track by $index">{{ detail }}</li>
>> +    <li class="incorrect" ng-repeat="incorrDetail in incorrDetails track by $index">{{ incorrDetail }}</li>
> What does the track by $index do?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/HarmoniousWombat/HarmoniousWombat/pull/67/files#r26727343